stages/org.osbuild.sysconfig

```diff
@@ -164,6 +164,20 @@ SCHEMA = r"""
         "description": "Livesys session to use"
       }
     }
+  },
+  "desktop": {
+    "additionalProperties": false,
+    "type": "object",
+    "properties": {
+      "preferred": {
+        "type": "string",
+        "description": "Preferred desktop to use."
+      },
+      "displaymanager": {
+        "type": "string",
+        "description": "Displaymanager to use."
```

Comment:
I just realized that it would probably make sense to require that at least one property is specified...